src/Glpi/Config/LegacyConfigurators/StandardIncludes.php

```diff
@@ -219,17 +220,7 @@ public function execute(): void
             }
 
             if (!isset($_SESSION["glpiskipMaintenance"]) || !$_SESSION["glpiskipMaintenance"]) {
-                Session::loadLanguage('', false);
-                if (isCommandLine()) {
-                    echo __('Service is down for maintenance. It will be back shortly.');
-                    echo "\n";
-                } else {
```

Comment:
For me, the only place where this can be relevant in CLI mode is in the `/front/cron.php` script. A `try/catch` block should be added to surround `$kernel->loadCommonGlobalConfig();` in order to properly display the execption when this script is executed from the CLI.

Reply:
Done